package.json

```diff
@@ -114,6 +114,12 @@
     "@nuxt/typescript-build": "^2.1.0",
     "@nuxtjs/pwa": "^3.3.5",
     "@open-wc/webpack-import-meta-loader": "^0.4.7",
+    "@polkadot/api": "7.10.1",
+    "@polkadot/keyring": "^8.4.1",
+    "@polkadot/networks": "^8.4.1",
+    "@polkadot/ui-settings": "0.89.3",
+    "@polkadot/util": "^8.4.1",
+    "@polkadot/util-crypto": "^8.4.1",
```

Comment:
Not fan of this tbh.

Isn't it better to update the [@kodadot1 packages](https://github.com/kodadot/packages)

Reply:
sure, we can update `@kodadot1/minimark`. afaik, the rest of the packages still got the warning messages:
- vue-apollo -> graphql-tag
- @polkadot/extension-dapp. source: https://github.com/polkadot-js/extension/blob/master/packages/extension-dapp/package.json#L28-L32
  - @polkadot/api
  - @polkadot/util
  - @polkadot/util-crypto

etc...

there is an explicit message if the repo using `pnpm` 
<img width="936" alt="Screen Shot 2022-03-11 at 10 58 27 PM" src="https://user-images.githubusercontent.com/734428/158063354-4cc7987b-234d-47ca-900e-ddc5bdeda25c.png">

Reply:
for now, the app may be still running perfectly. but not good in the long run if we keep the warning. turns out:
- `apollo-boost` is deprecated https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#apollo-boost
- `@nuxtjs/apollo` is unmaintained and there is a security concern if I look at the details in snyk https://snyk.io/advisor/npm-package/@nuxtjs/apollo#security

those packages technically tested only in graphql <= 15, for now maybe still run with graphql 16. in the future, not sure still able to run with newer graphql or not